test/units/TEST-26-SYSTEMCTL.sh

```diff
@@ -520,4 +520,48 @@ systemctl daemon-reload
 systemctl enable --now test-WantedBy.service || :
 systemctl daemon-reload
 
+# Test systemctl edit --global and systemctl cat --global (issue #31272)
+GLOBAL_UNIT_NAME="systemctl-test-$RANDOM.service"
+GLOBAL_MASKED_UNIT="systemctl-test-masked-$RANDOM.service"
+
+# Test 1: Create a new global user unit with --force and --runtime
+systemctl edit --global --runtime --stdin --full --force "$GLOBAL_UNIT_NAME" <<EOF
```

Comment:
Hmm, so new feature… And potential for a regression… Is this really suitable for backporting?

Reply:
OK, I wasn't aware that it's solving a regression. It seems fine then.